### PR TITLE
Add "newsletters, etc" section

### PR DESCRIPTION
**Pull request by jonkiparsky**
Adding a section for newsletters, discussion boards, and other online information streams.

### CODE REVIEW

File: resources.md [hunk 1]
```diff
@@ -69,3 +63,15 @@ There are thousands of books on programming in Python. This list focuses on titl
...
+
+## Newsletters, mailing lists, discussion boards
+
+If you want to learn more about a language, listening to other people talk about it can be helpful. Discussion forums can give you new ideas and lead you into parts of the language that you wouldn't have otherwise discovered, and expose you to the ways other people think about and use the language.
+
+- [PyHerald](https://pyherald.com/) - A daily summary of what's happening on various Python forums, apparently sourced from official and unofficial Python mailing lists and discussion boards. Probably a good read for people who want to know what's going on in the Python world but don't have time to follow all of the lists themselves.
+
+- [python-ideas](https://mail.python.org/mailman3/lists/python-ideas.python.org/) - This list is mostly about proposals for future development of the Python language. Sometimes a bit contentious, sometimes a bit technical. Volume is typically moderate to low, but can surge if a particularly fraught proposal is made.
```
Comment (bhrutledge) [inline resources.md:73]: From the description, and what I've heard about python-ideas on newsletters and podcasts, it feels out of the scope of this page and its intended audience:

> This page lists a number of places you can go for ideas about what programs to write to develop your Python skills.

After a quick scan, I think the same might be true of PyHerald.

There are lots of Python-related newsletters, mailing lists, discussion boards, and podcasts. Are they useful to add to this page? How do we decide what makes the cut? I already feel like I'm bike-shedding. Maybe there's already a list of them somewhere, that we could link to as an additional resource?

File: resources.md [hunk 2]
```diff
@@ -65,12 +64,13 @@ There are thousands of books on programming in Python. This list focuses on titl
...
 
-- [PyHerald](https://pyherald.com/) - A daily summary of what's happening on various python fora, apparently sourced from official and unofficial python mailing lists and discussion boards. Probably a good read for people who want to know what's going on in the python world but don't have time to follow all of the lists themselves. 
-- [python-ideas](https://mail.python.org/mailman3/lists/python-ideas.python.org/) - This list is mostly about proposals for future development of the python language. Sometimes a bit contentious, sometimes a bit technical. Volume is typically moderate to low, but can surge if a particularly fraught proposal is made. 
+- [PyHerald](https://pyherald.com/) - A daily summary of what's happening on various Python forums, apparently sourced from official and unofficial Python mailing lists and discussion boards. Probably a good read for people who want to know what's going on in the Python world but don't have time to follow all of the lists themselves.
```
Comment (jonkiparsky) [inline resources.md:71]: Lol. We don't use nonstandard productive plura* here?  ;)

(In a related note, I've noticed recently that the Times is spelling the plural of "millennium" as "millenniums". Who am I to blow against the wind?)

*maybe the correct nonstandard productive plural of "plural" should be "plurais", by analogy to the Portuguese?

File: resources.md (hunk 1 above)
Comment (jonkiparsky) [reply resources.md:73]: My thinking in adding them was that they're good sources of curiosity for intermediate python programmers. They're probably not for everybody, but that's true of everything. 

Is there a quick fix you'd suggest to make this addition make more sense to you? If so, I'm happy to do that. If not, I'm fine with dropping this.

Comment (bhrutledge) [reply resources.md:73]: I'm inclined to drop it for now.

**Pull request closed without merging**